client/src/Components/forms/UserProfileUpdateForm.js

```diff
@@ -18,15 +18,15 @@ const UserProfileUpdateForm = (props) => {
                     passwordConfirm: response.data[0].password
```

Comment:
I think we will have the same issue as the others that the password will return a hash.

Reply:
fixed this in the last commit

Reply:
👍